projects/timer_challenge/src/main.c

```diff
@@ -0,0 +1,37 @@
+#include <stdint.h>
+#include <stdlib.h>
```

Comment:
I don't think we need stdlib here? Though I may be wrong
Try removing it and see if it still runs properly